### PR TITLE
Feature/6-Simplify-environment-installation

### PR DESCRIPTION
**Pull request by IKostric**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+pyowm
+selenium
+webdriver-manager
+pycountry
+pandas
+fuzzywuzzy
+git+https://github.com/LIAAD/yake
+countryinfo
+zmq
+face_recognition
```
Comment (NoB0) [inline requirements.txt]: Nit: Is this package necessary as we do not have the face recognition feature?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Levenshtein
```
Comment (NoB0) [inline requirements.txt]: Nit: Is this package necessary?

**Review by NoB0 — APPROVED** [review]
LGTM. Just some comments to check for the environment. Please fix linting before merging.

**Review by WerLaj — APPROVED** [review]
LGTM

File: requirements.txt (hunk 1 above)
Comment (IKostric) [reply requirements.txt]: Removed

**Pull request merged**